### PR TITLE
adding LC description to Request a Workshop page

### PR DESCRIPTION
**Pull request by tracykteal**
Added description of Library Carpentry workshops to the new Request a Carpentries Workshop page.

### CODE REVIEW

File: workshops/models.py [hunk 1]
```diff
@@ -2759,6 +2759,9 @@ class WorkshopRequest(AssignmentMixin, StateMixin, CreatedUpdatedMixin,
         "<a href='http://www.datacarpentry.org/lessons/'>"
         "Data Carpentry lessons page</a>"
     )
+    LC_LESSONS_LINK = (
+        "<a href='https://librarycarpentry.org/'>"
+        "Library Carpentry</a>"
```
Comment (pbanaszkiewicz) [inline workshops/models.py]:
```suggestion
        "Library Carpentry</a>"
    )
```

This fixes syntax error.

File: workshops/models.py [hunk 2]
```diff
@@ -2768,8 +2771,11 @@ class WorkshopRequest(AssignmentMixin, StateMixin, CreatedUpdatedMixin,
                   "the best choice. If your learners are interested in "
                   "learning more about programming, including version control"
                   " and automation, Software Carpentry is likely the best "
-                  "match. Please visit the " + SWC_LESSONS_LINK + " or the "
-                  + DC_LESSONS_LINK +
+                  "match. If your learners are people working in library and "
+                  "information related roles interested in learning data and "
+                  "software skills, Library Carpentry is the best choice. " 
+                  "Please visit the " + SWC_LESSONS_LINK + " or the "
```
Comment (pbanaszkiewicz) [inline workshops/models.py]:
```suggestion
                  "Please visit the " + SWC_LESSONS_LINK + ", "
```

File: workshops/models.py [hunk 3]
```diff
@@ -2768,8 +2771,11 @@ class WorkshopRequest(AssignmentMixin, StateMixin, CreatedUpdatedMixin,
...
+                  + DC_LESSONS_LINK + LC_LESSONS_LINK +
```
Comment (pbanaszkiewicz) [inline workshops/models.py]:
```suggestion
                  + DC_LESSONS_LINK + ", or the " + LC_LESSONS_LINK +
```

**Comment by tracykteal** [comment]
Thanks so much @pbanaszkiewicz. Really appreciate the help. I accepted those suggestions and made those updates.

**Comment by pbanaszkiewicz** [comment]
I believe these changes trigger a migration, so I'll take it from here and add it if indeed required.

**Comment by tracykteal** [comment]
Great thank you. I was wondering what the timeline would be to get it so that it's deployed and live on the workshop request form.

**Comment by pbanaszkiewicz** [comment]
@tracykteal it should be live now!

**Comment by tracykteal** [comment]
Great, thank you!

**Pull request merged**